iredis/client.py

```diff
@@ -407,12 +407,7 @@ def send_command(self, raw_command, completer=None):  # noqa
             # subcommand's stdout/stderr
             if shell_command and config.shell:
                 # pass the raw response of redis to shell command
-                if isinstance(redis_resp, list):
-                    # FIXME not handling nested list, use renders.render_raw
-                    # instead
-                    stdin = b"\n".join(redis_resp)
-                else:
-                    stdin = redis_resp
+                stdin = OutputRender.render_raw(redis_resp)
```

Comment:
`List` also has then same problem when use `decode` option. We need to flatten the nested `list`, so I add additional type Conversion in `render_raw`.